src/components/editions/header.tsx

```diff
@@ -77,16 +77,24 @@ const galleryHeaderBorderStyles = css`
 	}
 `;
 
-const interviewHeaderStyles = (item: Format): SerializedStyles => css`
-	${from.tablet} {
-		padding-left: ${tabletArticleMargin}px;
-	}
+const interviewHeaderStyles = (item: Item): SerializedStyles => {
```

Comment:
Should the name be updated here now it's not being used exclusively for interview?

Reply:
Good idea!